opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/dto/result/torest/redirectable/FacadeResultRedirectable.java

```diff
@@ -16,6 +16,7 @@
     private UUID xRequestId;
     private String serviceSessionId;
     private String authorizationSessionId;
+    private String code;
```

Comment:
Suggest to name it as `oauth2Code` for explicitness (we can't do it in API, but we can here)